assets/js/ajax-filters.js

```diff
@@ -175,15 +183,13 @@ jQuery( document ).ready( function ( $ ) {
 		} );
 	} );
 
-	$( '#search_keywords, #search_location, .job_types :input, #search_categories, .job-manager-filter' ).change( function() {
-		var target   = $( this ).closest( 'div.job_listings' );
-		target.triggerHandler( 'update_results', [ 1, false ] );
-		job_manager_store_state( target, 1 );
-	} )
+	//end of the initial update_results listener
 
-	.on( "keyup", function(e) {
+	// Changed this to just listen for enter button instead of listening for change event which was calling the loop twice.
```

Comment:
further to my last comments the reason the form was submitting twice was because of implicit submission on the enter key I opened [a question about this on stackoverflow](http://stackoverflow.com/questions/33026415/keyup-13-event-repeats-implied-submission-submit-form-on-enter-issue) and have now fixed the issue by preventing default on the keypress 13 event.